docs/tutorial.md

```diff
@@ -0,0 +1,295 @@
+# DNS-based discovery Tree Creation: Basic Tutorial
+
+## Background
+
+The `tree_creator` is a command line utility used to create and update [EIP-1459](https://eips.ethereum.org/EIPS/eip-1459) compliant Merkle trees.
+It takes as input a list of node records (in ENR text encoding) and link entries pointing to other trees.
```

Comment:
What does "link" mean? In this context? What is the output/result?

Reply:
These are link entries in the form `enrtree://` followed by a URL to another deployed node list at another domain. This can be used to traverse several node list trees recursively when discovering nodes, link to existing deployed lists, etc.